subito-searcher.py

```diff
@@ -63,37 +64,74 @@ def run_query(url, name):
     page = requests.get(url)
     soup = BeautifulSoup(page.text, 'html.parser')
 
-    products_list = soup.find(class_='items_listing')
-    product_list_items = products_list.find_all(class_='item_list_inner')
+    # find the father of the visible items
+    text = re.findall("jsx-[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9] items visible", page.text)
```

Comment:
Ok, I get it, you want to be less inclusive with the regex. But doing it this way is slightly redunant and does not adapt well to changes. wouldn't be "jsx-[0-9]* items visible" more flexible and clean?

Reply:
Yes. you are right. the source code was written very quickly. in first time, the regex didn't work, but perhaps i was the error ;)